src/alea.ts

```diff
@@ -0,0 +1,20 @@
+import { Alea } from "./deps.ts";
+
+export interface Alea {
+  (): number;
+  importState(state: number[]): void;
+  exportState(): number[];
+}
+
+export function createAlea(seed?: string) {
+  return (seed ? new Alea(seed) : new Alea()) as Alea;
+}
+
+export function isAlea(value: unknown): value is Alea {
```

Comment:
This does not appear to be used anywhere.  Can we remove it?  I'm not a big fan of type guards because you are telling the compiler what something is rather than the other way around

Reply:
Death to unused code!